dataland-backend/src/test/kotlin/org/dataland/datalandbackend/services/CompanyManagerTest.kt

```diff
@@ -47,10 +48,12 @@ class CompanyManagerTest(
 
     @Test
     fun `retrieve companies as a list and check for each company if it can be found as expected`() {
-        val allCompaniesInStore = testCompanyManager.searchCompaniesAndGetApiModel(CompanySearchFilter())
+        val allCompaniesInStore = testCompanyManager.searchCompaniesByNameOrIdentifierAndGetApiModel(
+            "",
+        )
         assertTrue(
             allCompaniesInStore.all {
-                testCompanyList.any { testCompany -> testCompany.companyName == it.companyInformation.companyName }
+                testCompanyList.any { testCompany -> testCompany.companyName == it.companyName }
             },
             "Not all the companyInformation of the posted companies could be found in the stored companies.",
```

Comment:
Misleading fail message.   Should be that not all the company names of the posted companies could be found in the stored companies.   There is no assertion on the whole companyInformation objets.

Reply:
The assertion message is actually correct. If there is one name missing, then the whole information is missing.

Reply:
Das ist ein Schluss, den du aus dem fehlenden companyName ziehst, aber faktisch failt der Test einfach nur, wenn der Name nicht gefunden werden kann.
Aber wir können es auch gerne lassen.